### PR TITLE
Add sanity check commands to TEtranscripts-2.2.0-foss-2020a easyconfig

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/t/TEtranscripts/TEtranscripts-2.2.0-foss-2020a.eb [hunk 1]
```diff
@@ -34,4 +34,6 @@ sanity_check_paths = {
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
 }
 
+sanity_check_commands = [('TEtranscripts', '--version'), ('TEcount', '--version')]
```
Comment (boegel) [inline easybuild/easyconfigs/t/TEtranscripts/TEtranscripts-2.2.0-foss-2020a.eb:37]: @hajgato No need for tuples, can be just string values with the full command, and we generally use on entry per line:

```suggestion
sanity_check_commands = [
    "TEtranscripts --version",
    "TEcount --version",
]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3406.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/4072ae9e056b274d9143fea0edf536e6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2441.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/d4fcfaf024bf10369e2f378c7a83925c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/eb1a94d45fc72d404e2017b428cf1065 for a full test report.

**Comment by hajgato** [comment]
can someone re-trigger the python 3 travis check? Or tell me how to do that?

**Comment by verdurin** [comment]
@hajgato I just closed and re-opened, which has that effect.

**Comment by hajgato** [comment]
ah, so this is the old-school close-reopen :)

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**